packages/react/src/components/Forms/Input/_index.scss

```diff
@@ -133,3 +133,10 @@
     }
   }
 }
+
+.c__input--password {
+  .c__input__icon-right .material-icons {
+    font-size: var(--c--components--forms-input--password-icon-size);
+  }
+}
```

Comment:
I wonder if we should allow to customize icon size that way.
IMO we should allow to customize size of leftIcon and rightIcon. But customize the size of the password input show/hide icon looks too specific. WDYT ?

Reply:
I agree but if we don't do that it looks really weird

## Current

<img width="359" alt="Capture d’écran 2024-03-19 à 16 01 32" src="https://github.com/openfun/cunningham/assets/9628870/e5364be9-b118-4a12-a731-e6508b43cdd2">

## Small 

<img width="367" alt="Capture d’écran 2024-03-19 à 16 01 47" src="https://github.com/openfun/cunningham/assets/9628870/6c43c2ae-0220-4c7b-842c-08a421a11314">

## Medium 

> Notice the weird padding right

<img width="358" alt="Capture d’écran 2024-03-19 à 16 02 07" src="https://github.com/openfun/cunningham/assets/9628870/677a0f35-b1b7-4901-9f59-fb7026d5b422">

Reply:
Oh right I get the problem. The button width is 48px so its larger than if we were using a classic icon.

Reply:
All right, Hagrid